doozerlib/distgit.py

```diff
@@ -1821,8 +1821,11 @@ def _merge_source(self):
         else:
             dockerfile_name = "Dockerfile"
 
-        # The path to the source Dockerfile we are reconciling against
-        source_dockerfile_path = os.path.join(self.source_path(), dockerfile_name)
+        # The path to the source Dockerfile we are reconciling against. Resolve to actual file
+        # if it is a symlink.
+        source_dockerfile_path = os.path.relpath(os.path.join(self.source_path(), dockerfile_name))
```

Comment:
Shouldn't this be `os.path.realpath()`?  `os.path.relpath` isn't documented as doing anything with symlinks.

Reply:
Just `a` small difference in names there.

Reply:
Thanks, @cgwalters !